playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -30,6 +30,14 @@
       - tcpdump
       etcd_ip: ETCD_IP_REPLACE
 
+  - name: set rhel dev facts
```

Comment:
can you get rid of this stanza and update https://github.com/openshift/aos-cd-jobs/pull/1518 such that it just calls the playbook with `-e oreg_url='registry.reg-aws.openshift.com/openshift3/ose-${component}:${version}' -e oreg_auth_user=unused -e oreg_auth_password='...'` ?